Dockerfile

```diff
@@ -12,16 +12,9 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
 # Airflow
-# install from source|pip ?
-ARG INSTALL_FROM=pip
-ARG AIRFLOW_VERSION=1.9.0
-ARG AIRFLOW_EXTRAS=async,celery,crypto,jdbc,hdfs,hive,azure,gcp_api,emr,password,postgres,slack,ssh,mysql
```

Comment:
would prefer the AIRFLOW_EXTRAS via  ARG so that it could be parameterized by docker file instead of cloning the repo.